src/Microsoft.AspNetCore.Server.Kestrel/Http/FrameRequestStream.cs

```diff
@@ -19,35 +19,51 @@ public FrameRequestStream()
             _state = FrameStreamState.Closed;
         }
 
-        public override bool CanRead { get { return true; } }
+        public override bool CanRead => true;
 
-        public override bool CanSeek { get { return false; } }
+        public override bool CanSeek => false;
```

Comment:
Why? Your suggestion adds a private backing field, which increases the cost in memory. The expression-bodied member just returns false, without adding a private field to the type.